tailwind.config.ts

```
@@ -1,35 +1,19 @@
 import type { Config } from 'tailwindcss';
+import { COLORS, SIZES, SCREENS as CUSTOMSCREENS } from './src/constants';
+
+const SCREENS: Record<string, string> = {};
+
+Object.entries(CUSTOMSCREENS).forEach((i) => (SCREENS[i[0]] = i[1] + 'px'));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)